testutils/lint: TestNightlyLint failed

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15826399?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15826399?buildTab=artifacts#/) on master @ [0e5781f3e2004ef6222e1ed4f5311b771936bd6a](https://github.com/cockroachdb/cockroach/commits/0e5781f3e2004ef6222e1ed4f5311b771936bd6a):


```
             1412:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             2187:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             2246:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             761:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             1651:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             376:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             248:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             824:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             2002:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             1946:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             1010:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             834:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             556:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             1215:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             1602:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             1506:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             1979:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             910:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             1889:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             1415:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             1315:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             421:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/create-function.html : 404 Not Found
             2125:  https://www.cockroachlabs.com/docs/v24.2/create-function.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             959:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             2023:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             2188:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             108:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (1.45s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-39807

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15840906?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15840906?buildTab=artifacts#/) on master @ [d2a5af43be05ade40531e6e0ef23c5880faa4494](https://github.com/cockroachdb/cockroach/commits/d2a5af43be05ade40531e6e0ef23c5880faa4494):


```
             67:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             739:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             1226:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/backup.html : 404 Not Found
             1963:  https://www.cockroachlabs.com/docs/v24.2/backup.html
        https://www.cockroachlabs.com/docs/v24.2/delete.html : 404 Not Found
             939:  https://www.cockroachlabs.com/docs/v24.2/delete.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             51:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             1595:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             658:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             2250:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             1395:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             2156:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             2166:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/create-function.html : 404 Not Found
             1539:  https://www.cockroachlabs.com/docs/v24.2/create-function.html
        https://www.cockroachlabs.com/docs/v24.2/table-expressions.html : 404 Not Found
             525:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             814:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1063:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1222:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1501:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1569:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1695:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             815:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             368:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             625:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             231:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             1441:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             130:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1858:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             1125:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.65s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15853577?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15853577?buildTab=artifacts#/) on master @ [b4402507fdebe5682f77e348331a308d378c91e1](https://github.com/cockroachdb/cockroach/commits/b4402507fdebe5682f77e348331a308d378c91e1):


```
        https://www.cockroachlabs.com/docs/v24.2/set-vars.html : 404 Not Found
             355:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1038:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1355:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1772:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             2010:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             1445:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/backup.html : 404 Not Found
             756:  https://www.cockroachlabs.com/docs/v24.2/backup.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             1648:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             915:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             1821:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             433:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             461:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             404:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/table-expressions.html : 404 Not Found
             70:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             139:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             177:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             895:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1022:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1167:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1441:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             671:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             935:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             1870:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             1446:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             1459:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             2000:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             1804:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             2184:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             2147:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15867126?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15867126?buildTab=artifacts#/) on master @ [047a7ed79756eef53b8b9ab4c9dd9c5a463496c9](https://github.com/cockroachdb/cockroach/commits/047a7ed79756eef53b8b9ab4c9dd9c5a463496c9):


```
             30:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             2087:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             1482:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/delete.html : 404 Not Found
             936:  https://www.cockroachlabs.com/docs/v24.2/delete.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             2204:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             1087:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             520:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1717:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             1529:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/drop-database.html : 404 Not Found
             1652:  https://www.cockroachlabs.com/docs/v24.2/drop-database.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             907:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        https://www.cockroachlabs.com/docs/v24.2/create-function.html : 404 Not Found
             791:  https://www.cockroachlabs.com/docs/v24.2/create-function.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             1973:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             461:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             612:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.2/alter-view.html : 404 Not Found
             1201:  https://www.cockroachlabs.com/docs/v24.2/alter-view.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             2020:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             1541:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             669:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             2030:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             1895:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             81:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             828:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             1483:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             413:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (2.81s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15875232?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15875232?buildTab=artifacts#/) on master @ [d13253527955eaa2da09394b8a2729627ab25c48](https://github.com/cockroachdb/cockroach/commits/d13253527955eaa2da09394b8a2729627ab25c48):


```
             219:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             1161:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             1568:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             156:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             1845:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             245:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             593:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             618:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             375:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             2250:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             495:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             1935:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             1112:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             954:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             2001:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             1934:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/create-function.html : 404 Not Found
             1540:  https://www.cockroachlabs.com/docs/v24.2/create-function.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             79:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             2109:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             2073:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             88:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             796:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             2019:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             869:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/drop-database.html : 404 Not Found
             171:  https://www.cockroachlabs.com/docs/v24.2/drop-database.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.47s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15882737?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15882737?buildTab=artifacts#/) on master @ [d13253527955eaa2da09394b8a2729627ab25c48](https://github.com/cockroachdb/cockroach/commits/d13253527955eaa2da09394b8a2729627ab25c48):


```
             1451:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             1722:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             248:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             2054:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             375:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             545:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             760:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/table-expressions.html : 404 Not Found
             245:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             441:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1227:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1362:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1531:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1660:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1798:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             2026:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             1094:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             985:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             1028:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             2250:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             940:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             155:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             520:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             1843:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             1833:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        https://www.cockroachlabs.com/docs/v24.2/drop-database.html : 404 Not Found
             1977:  https://www.cockroachlabs.com/docs/v24.2/drop-database.html
        https://www.cockroachlabs.com/docs/v24.2/show-vars.html : 404 Not Found
             350:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
             2174:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             505:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1396:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.53s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16171805?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16171805?buildTab=artifacts#/) on master @ [05052df0b4d89dd1147714431163cd8a9b63c589](https://github.com/cockroachdb/cockroach/commits/05052df0b4d89dd1147714431163cd8a9b63c589):


```
I240721 05:14:43.050054 51 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html...
I240721 05:14:43.050109 46 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/alter-index.html...
I240721 05:14:43.050174 53 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/delete.html...
I240721 05:14:43.050247 52 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/truncate.html...
I240721 05:14:43.050320 50 (gostd) urlcheck.go:235  [-] 18  Checking https://www.cockroachlabs.com/docs/v24.2/explain.html...
I240721 05:14:43.050381 54 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/insert.html...
I240721 05:14:43.050456 45 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.2/import-into.html...
I240721 05:14:43.071055 202 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/show-create.html...
I240721 05:14:43.071147 174 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/alter-partition.html...
I240721 05:14:43.071389 244 (gostd) urlcheck.go:235  [-] 23  Checking https://en.wikipedia.org/wiki/Cron...
I240721 05:14:43.071457 175 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.2/alter-table.html...
I240721 05:14:43.071591 216 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html...
I240721 05:14:43.074426 184 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/show-databases.html...
I240721 05:14:43.082241 187 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/drop-table.html...
I240721 05:14:43.178909 221 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/show-vars.html...
I240721 05:14:43.181186 188 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/show-default-privileges...
I240721 05:14:43.181465 222 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/alter-function.html...
I240721 05:14:43.182335 189 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/create-table-as.html...
I240721 05:14:43.183665 223 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.2/create-procedure.html...
I240721 05:14:43.184580 190 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/create-trigger.html...
I240721 05:14:43.191980 224 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/savepoint.html...
I240721 05:14:43.193017 191 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/show-index.html...
I240721 05:14:43.193087 225 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html...
I240721 05:14:43.193372 203 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240721 05:14:43.194309 258 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/table-expressions.html...
I240721 05:14:43.195909 204 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/alter-database.html...
I240721 05:14:43.197579 259 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html...
I240721 05:14:43.197651 205 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html...
I240721 05:14:43.199192 192 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/create-database.html...
I240721 05:14:43.199971 206 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.2/set-transaction.html...
I240721 05:14:43.201500 193 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/show-tables.html...
I240721 05:14:43.202228 207 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/show-users.html...
I240721 05:14:43.202380 208 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.2/show-backup.html...
I240721 05:14:43.212351 274 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/revoke.html...
I240721 05:14:43.213371 209 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html...
I240721 05:14:43.217917 275 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/set-vars.html...
I240721 05:14:43.221265 290 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/show-ranges.html...
I240721 05:14:43.236757 276 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/upsert.html...
I240721 05:14:43.245907 291 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/select-clause.html...
I240721 05:14:43.300872 277 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.2/restore.html...
I240721 05:14:43.301962 292 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.2/drop-database.html...
I240721 05:14:43.302392 278 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/drop-function.html...
I240721 05:14:43.305111 279 (gostd) urlcheck.go:235  [-] 56  Checking https://www.cockroachlabs.com/docs/v24.2/drop-index.html...
I240721 05:14:43.314095 293 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/alter-type.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             1479:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             845:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16180367?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16180367?buildTab=artifacts#/) on master @ [05052df0b4d89dd1147714431163cd8a9b63c589](https://github.com/cockroachdb/cockroach/commits/05052df0b4d89dd1147714431163cd8a9b63c589):


```
I240722 05:14:15.599019 63 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/show-tables.html...
I240722 05:14:15.599122 65 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/create-index.html...
I240722 05:14:15.599185 64 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/select-clause.html...
I240722 05:14:15.599241 50 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/create-database.html...
I240722 05:14:15.599351 67 (gostd) urlcheck.go:235  [-] 18  Checking https://www.cockroachlabs.com/docs/v24.2/alter-database.html...
I240722 05:14:15.599423 62 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/set-vars.html...
I240722 05:14:15.599500 51 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.2/grant.html...
I240722 05:14:15.614161 186 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/backup.html...
I240722 05:14:15.614927 160 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/import-into.html...
I240722 05:14:15.615566 187 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.2/create-procedure.html...
I240722 05:14:15.616148 161 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html...
I240722 05:14:15.616204 188 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/alter-function.html...
I240722 05:14:15.616462 242 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/show-users.html...
I240722 05:14:15.616974 189 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/drop-table.html...
I240722 05:14:15.617046 243 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/revoke.html...
I240722 05:14:15.617329 190 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/show-databases.html...
I240722 05:14:15.617542 244 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/alter-table.html...
I240722 05:14:15.618473 191 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html...
I240722 05:14:15.619937 245 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.2/delete.html...
I240722 05:14:15.620008 258 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/savepoint.html...
I240722 05:14:15.620729 246 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/show-default-privileges...
I240722 05:14:15.621780 259 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/alter-type.html...
I240722 05:14:15.622654 247 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/show-backup.html...
I240722 05:14:15.623069 260 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html...
I240722 05:14:15.625555 248 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html...
I240722 05:14:15.626045 261 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240722 05:14:15.626814 249 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/upsert.html...
I240722 05:14:15.627391 262 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/restore.html...
I240722 05:14:15.630902 250 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/create-table.html...
I240722 05:14:15.631004 263 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.2/alter-index.html...
I240722 05:14:15.631614 229 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/show-ranges.html...
I240722 05:14:15.632349 264 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/truncate.html...
I240722 05:14:15.635035 230 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.2/set-transaction.html...
I240722 05:14:15.636858 265 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/drop-database.html...
I240722 05:14:15.639617 231 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/insert.html...
I240722 05:14:15.640840 266 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/alter-view.html...
I240722 05:14:15.646254 232 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/update.html...
I240722 05:14:15.649546 267 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/show-create.html...
I240722 05:14:15.653647 233 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/create-function.html...
I240722 05:14:15.730919 268 (gostd) urlcheck.go:235  [-] 53  Checking https://en.wikipedia.org/wiki/Cron...
I240722 05:14:15.731002 234 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.2/show-columns.html...
I240722 05:14:15.735627 251 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/show-partitions.html...
I240722 05:14:15.735948 192 (gostd) urlcheck.go:235  [-] 56  Checking https://www.cockroachlabs.com/docs/v24.2/create-view.html...
I240722 05:14:15.739999 275 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/drop-index.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             1544:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             1470:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.33s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16199257?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16199257?buildTab=artifacts#/) on master @ [4af6b13bf95a52fe8f6857ffd32e8c1156c36950](https://github.com/cockroachdb/cockroach/commits/4af6b13bf95a52fe8f6857ffd32e8c1156c36950):


```
I240724 05:15:32.901767 67 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html...
I240724 05:15:32.901812 65 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/drop-index.html...
I240724 05:15:32.901882 56 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/savepoint.html...
I240724 05:15:32.901936 66 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/delete.html...
I240724 05:15:32.902117 63 (gostd) urlcheck.go:235  [-] 18  Checking https://www.cockroachlabs.com/docs/v24.2/insert.html...
I240724 05:15:32.902198 62 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/revoke.html...
I240724 05:15:32.902272 68 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.2/alter-index.html...
I240724 05:15:33.061298 217 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html...
I240724 05:15:33.062989 113 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/show-databases.html...
I240724 05:15:33.203937 220 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.2/create-trigger.html...
I240724 05:15:33.210204 184 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.2/create-procedure.html...
I240724 05:15:33.211199 221 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/upsert.html...
I240724 05:15:33.211845 185 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/show-users.html...
I240724 05:15:33.231034 222 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html...
I240724 05:15:33.236171 186 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/show-columns.html...
I240724 05:15:33.236680 223 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/import-into.html...
I240724 05:15:33.236777 187 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/show-default-privileges...
I240724 05:15:33.238031 264 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/set-transaction.html...
I240724 05:15:33.238544 188 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.2/show-backup.html...
I240724 05:15:33.238857 265 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/show-vars.html...
I240724 05:15:33.239191 189 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/drop-function.html...
I240724 05:15:33.256671 266 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html...
I240724 05:15:33.267060 190 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/set-vars.html...
I240724 05:15:33.269579 267 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.2/create-table-as.html...
I240724 05:15:33.272216 191 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/create-function.html...
I240724 05:15:33.280322 268 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/show-index.html...
I240724 05:15:33.327481 192 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/show-partitions.html...
I240724 05:15:33.357171 224 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/alter-database.html...
I240724 05:15:33.364358 273 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/truncate.html...
I240724 05:15:33.373605 225 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.2/show-ranges.html...
I240724 05:15:33.470537 290 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/alter-table.html...
I240724 05:15:33.490173 306 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/update.html...
I240724 05:15:33.499369 291 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.2/alter-function.html...
I240724 05:15:33.503153 293 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/show-grants.html...
I240724 05:15:33.503219 292 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/grant.html...
I240724 05:15:33.514303 307 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/select-clause.html...
I240724 05:15:33.516728 279 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html...
I240724 05:15:33.521616 308 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/drop-database.html...
I240724 05:15:33.525860 280 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/create-database.html...
I240724 05:15:33.527742 309 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240724 05:15:33.533592 281 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.2/drop-table.html...
I240724 05:15:33.540468 310 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/show-create.html...
I240724 05:15:33.542130 282 (gostd) urlcheck.go:235  [-] 56  Checking https://en.wikipedia.org/wiki/Cron...
I240724 05:15:33.544865 287 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/alter-type.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             1698:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             1365:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.98s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16208828?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16208828?buildTab=artifacts#/) on master @ [da85c982fa391ef34fd14c29006b1f629217ff42](https://github.com/cockroachdb/cockroach/commits/da85c982fa391ef34fd14c29006b1f629217ff42):


```
I240725 05:14:05.565300 15 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/create-database.html...
I240725 05:14:05.565341 74 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html...
I240725 05:14:05.565390 77 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html...
I240725 05:14:05.565424 72 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/create-trigger.html...
I240725 05:14:05.565474 76 (gostd) urlcheck.go:235  [-] 18  Checking https://www.cockroachlabs.com/docs/v24.2/show-backup.html...
I240725 05:14:05.565510 70 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/show-grants.html...
I240725 05:14:05.565559 78 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.2/insert.html...
I240725 05:14:05.720755 193 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/grant.html...
I240725 05:14:05.733464 226 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/alter-index.html...
I240725 05:14:05.733632 142 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.2/import-into.html...
I240725 05:14:05.737802 229 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.2/restore.html...
I240725 05:14:05.742227 245 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html...
I240725 05:14:05.757412 246 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/drop-index.html...
I240725 05:14:05.858016 238 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/select-clause.html...
I240725 05:14:05.861808 247 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/explain.html...
I240725 05:14:05.867602 239 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/create-function.html...
I240725 05:14:05.869275 248 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/show-tables.html...
I240725 05:14:05.878005 240 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/truncate.html...
I240725 05:14:05.881210 249 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.2/show-databases.html...
I240725 05:14:05.884341 241 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/alter-partition.html...
I240725 05:14:05.884635 250 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/show-ranges.html...
I240725 05:14:05.887054 258 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html...
I240725 05:14:05.891141 251 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html...
I240725 05:14:05.897908 259 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.2/table-expressions.html...
I240725 05:14:05.903364 252 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/set-vars.html...
I240725 05:14:05.908904 260 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/update.html...
I240725 05:14:05.923758 253 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/show-index.html...
I240725 05:14:05.931210 261 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/show-create.html...
I240725 05:14:05.979117 254 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/create-table.html...
I240725 05:14:05.995449 262 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240725 05:14:06.005564 263 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html...
I240725 05:14:06.010769 255 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/create-index.html...
I240725 05:14:06.011828 256 (gostd) urlcheck.go:235  [-] 46  Checking https://en.wikipedia.org/wiki/Cron...
I240725 05:14:06.013107 277 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/drop-database.html...
I240725 05:14:06.023570 280 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/alter-function.html...
I240725 05:14:06.045146 291 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/upsert.html...
I240725 05:14:06.063728 281 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/show-columns.html...
I240725 05:14:06.093020 292 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/create-procedure.html...
I240725 05:14:06.099990 282 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/drop-table.html...
I240725 05:14:06.108467 293 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.2/create-view.html...
I240725 05:14:06.132621 283 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.2/savepoint.html...
I240725 05:14:06.141330 294 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/show-users.html...
I240725 05:14:06.145454 284 (gostd) urlcheck.go:235  [-] 56  Checking https://www.cockroachlabs.com/docs/v24.2/show-default-privileges...
I240725 05:14:06.146554 295 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/create-table-as.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             2194:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             1352:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.88s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16217305?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16217305?buildTab=artifacts#/) on master @ [7fb362dd5aa6e85d65c4c89f208c5bed51ab9692](https://github.com/cockroachdb/cockroach/commits/7fb362dd5aa6e85d65c4c89f208c5bed51ab9692):


```
I240726 05:14:24.974169 58 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html...
I240726 05:14:24.974292 59 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/insert.html...
I240726 05:14:24.974363 57 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/alter-index.html...
I240726 05:14:24.974410 25 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/alter-view.html...
I240726 05:14:24.974492 53 (gostd) urlcheck.go:235  [-] 18  Checking https://www.cockroachlabs.com/docs/v24.2/show-index.html...
I240726 05:14:24.974531 55 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/savepoint.html...
I240726 05:14:24.974589 31 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.2/drop-index.html...
I240726 05:14:25.142197 145 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/show-default-privileges...
I240726 05:14:25.262303 244 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html...
I240726 05:14:25.265949 230 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.2/show-users.html...
I240726 05:14:25.269455 245 (gostd) urlcheck.go:235  [-] 24  Checking https://en.wikipedia.org/wiki/Cron...
I240726 05:14:25.270982 231 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/alter-type.html...
I240726 05:14:25.277927 234 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/backup.html...
I240726 05:14:25.278951 254 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html...
I240726 05:14:25.279013 235 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/show-databases.html...
I240726 05:14:25.279314 220 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/show-columns.html...
I240726 05:14:25.285528 236 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/delete.html...
I240726 05:14:25.286138 221 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html...
I240726 05:14:25.286604 237 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.2/create-view.html...
I240726 05:14:25.294076 222 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/import-into.html...
I240726 05:14:25.300686 238 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/explain.html...
I240726 05:14:25.303535 223 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/show-partitions.html...
I240726 05:14:25.307150 239 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/restore.html...
I240726 05:14:25.308468 224 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.2/create-database.html...
I240726 05:14:25.309586 240 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/create-table.html...
I240726 05:14:25.310524 225 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240726 05:14:25.310572 241 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/alter-partition.html...
I240726 05:14:25.326780 255 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html...
I240726 05:14:25.402565 258 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/create-table-as.html...
I240726 05:14:25.413375 263 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html...
I240726 05:14:25.454663 292 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/create-function.html...
I240726 05:14:25.546610 264 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/set-transaction.html...
I240726 05:14:25.554773 293 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html...
I240726 05:14:25.559543 265 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/upsert.html...
I240726 05:14:25.560401 294 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/create-index.html...
I240726 05:14:25.563327 266 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/show-ranges.html...
I240726 05:14:25.565484 295 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/alter-table.html...
I240726 05:14:25.574108 267 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/grant.html...
I240726 05:14:25.575725 296 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/create-procedure.html...
I240726 05:14:25.575967 268 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.2/show-grants.html...
I240726 05:14:25.580900 297 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.2/alter-database.html...
I240726 05:14:25.597573 269 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/show-tables.html...
I240726 05:14:25.605484 298 (gostd) urlcheck.go:235  [-] 56  Checking https://www.cockroachlabs.com/docs/v24.2/drop-database.html...
I240726 05:14:25.615440 270 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/revoke.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             166:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             950:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.95s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16229831?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16229831?buildTab=artifacts#/) on master @ [1893ac1d7188dc87dbb2b379b30172bf83ca4645](https://github.com/cockroachdb/cockroach/commits/1893ac1d7188dc87dbb2b379b30172bf83ca4645):


```
I240727 05:14:33.972003 88 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/drop-table.html...
I240727 05:14:33.972045 47 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/show-create.html...
I240727 05:14:33.972120 43 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/alter-view.html...
I240727 05:14:33.972163 48 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/show-partitions.html...
I240727 05:14:33.972249 42 (gostd) urlcheck.go:235  [-] 18  Checking https://en.wikipedia.org/wiki/Cron...
I240727 05:14:33.972353 44 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html...
I240727 05:14:33.972469 45 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.2/select-clause.html...
I240727 05:14:33.982276 101 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/drop-database.html...
I240727 05:14:34.114331 259 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/grant.html...
I240727 05:14:34.120232 159 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.2/create-function.html...
I240727 05:14:34.120461 260 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html...
I240727 05:14:34.122546 245 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/create-trigger.html...
I240727 05:14:34.123466 261 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/alter-function.html...
I240727 05:14:34.123677 246 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/create-database.html...
I240727 05:14:34.124758 262 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/savepoint.html...
I240727 05:14:34.125470 247 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/show-databases.html...
I240727 05:14:34.126228 263 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/show-index.html...
I240727 05:14:34.127019 248 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/import-into.html...
I240727 05:14:34.128694 264 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.2/alter-index.html...
I240727 05:14:34.129569 265 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html...
I240727 05:14:34.135080 249 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/create-table.html...
I240727 05:14:34.135736 266 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html...
I240727 05:14:34.152320 250 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240727 05:14:34.155191 267 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.2/create-table-as.html...
I240727 05:14:34.167542 251 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/explain.html...
I240727 05:14:34.169503 268 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/set-vars.html...
I240727 05:14:34.191270 252 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/show-users.html...
I240727 05:14:34.197321 269 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/show-tables.html...
I240727 05:14:34.234664 253 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/drop-function.html...
I240727 05:14:34.239864 270 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html...
I240727 05:14:34.243208 254 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/show-vars.html...
I240727 05:14:34.244408 271 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/alter-partition.html...
I240727 05:14:34.245969 255 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.2/create-procedure.html...
I240727 05:14:34.246049 272 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html...
I240727 05:14:34.246817 160 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/delete.html...
I240727 05:14:34.247161 273 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/revoke.html...
I240727 05:14:34.252976 274 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/alter-table.html...
I240727 05:14:34.261625 275 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/drop-index.html...
I240727 05:14:34.264874 256 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html...
I240727 05:14:34.268122 284 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.2/upsert.html...
I240727 05:14:34.269864 190 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.2/show-backup.html...
I240727 05:14:34.275750 191 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/alter-type.html...
I240727 05:14:34.282975 309 (gostd) urlcheck.go:235  [-] 56  Checking https://www.cockroachlabs.com/docs/v24.2/show-default-privileges...
I240727 05:14:34.283974 192 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/show-grants.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             1228:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             943:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.51s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16236780?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16236780?buildTab=artifacts#/) on master @ [fb8032d12d2a787b54076812595e9d7fb22f925a](https://github.com/cockroachdb/cockroach/commits/fb8032d12d2a787b54076812595e9d7fb22f925a):


```
I240728 05:14:10.868120 60 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html...
I240728 05:14:10.868242 61 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html...
I240728 05:14:10.868321 69 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/update.html...
I240728 05:14:10.868366 57 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/alter-type.html...
I240728 05:14:10.868424 62 (gostd) urlcheck.go:235  [-] 18  Checking https://www.cockroachlabs.com/docs/v24.2/insert.html...
I240728 05:14:10.868507 58 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/alter-index.html...
I240728 05:14:10.868579 55 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.2/create-trigger.html...
I240728 05:14:10.885232 160 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/restore.html...
I240728 05:14:10.886295 242 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/set-transaction.html...
I240728 05:14:10.886613 161 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.2/explain.html...
I240728 05:14:10.886661 243 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.2/show-partitions.html...
I240728 05:14:10.886768 258 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/select-clause.html...
I240728 05:14:10.886837 244 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/show-databases.html...
I240728 05:14:10.886929 227 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/set-vars.html...
I240728 05:14:10.887338 197 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/show-create.html...
I240728 05:14:10.887686 245 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/create-view.html...
I240728 05:14:10.888847 198 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html...
I240728 05:14:10.890611 246 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/truncate.html...
I240728 05:14:10.891448 199 (gostd) urlcheck.go:235  [-] 32  Checking https://en.wikipedia.org/wiki/Cron...
I240728 05:14:10.892167 247 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/revoke.html...
I240728 05:14:10.896270 248 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/show-columns.html...
I240728 05:14:10.896859 206 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/drop-database.html...
I240728 05:14:10.896954 228 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/create-table.html...
I240728 05:14:10.897386 207 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.2/show-grants.html...
I240728 05:14:10.901464 208 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/show-default-privileges...
I240728 05:14:10.902856 249 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/drop-table.html...
I240728 05:14:11.002691 233 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/show-tables.html...
I240728 05:14:11.004273 275 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/grant.html...
I240728 05:14:11.005686 234 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/create-index.html...
I240728 05:14:11.005980 290 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.2/alter-table.html...
I240728 05:14:11.009508 276 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/show-ranges.html...
I240728 05:14:11.014632 299 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/delete.html...
I240728 05:14:11.015808 300 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.2/drop-function.html...
I240728 05:14:11.016432 263 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/create-database.html...
I240728 05:14:11.019912 301 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/alter-partition.html...
I240728 05:14:11.024553 264 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/backup.html...
I240728 05:14:11.029627 265 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/create-table-as.html...
I240728 05:14:11.030479 266 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240728 05:14:11.037582 302 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/table-expressions.html...
I240728 05:14:11.040338 267 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html...
I240728 05:14:11.045021 303 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.2/alter-view.html...
I240728 05:14:11.045626 268 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/show-backup.html...
I240728 05:14:11.050365 304 (gostd) urlcheck.go:235  [-] 56  Checking https://www.cockroachlabs.com/docs/v24.2/drop-index.html...
I240728 05:14:11.055097 269 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/create-procedure.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             214:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             2012:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.89s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16244044?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16244044?buildTab=artifacts#/) on master @ [fb8032d12d2a787b54076812595e9d7fb22f925a](https://github.com/cockroachdb/cockroach/commits/fb8032d12d2a787b54076812595e9d7fb22f925a):


```
I240729 05:14:21.167726 64 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/drop-database.html...
I240729 05:14:21.167761 56 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/show-grants.html...
I240729 05:14:21.167806 57 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/table-expressions.html...
I240729 05:14:21.167906 62 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/update.html...
I240729 05:14:21.167975 55 (gostd) urlcheck.go:235  [-] 18  Checking https://www.cockroachlabs.com/docs/v24.2/insert.html...
I240729 05:14:21.168029 61 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/show-index.html...
I240729 05:14:21.168082 58 (gostd) urlcheck.go:235  [-] 20  Checking https://en.wikipedia.org/wiki/Cron...
I240729 05:14:21.178162 120 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/alter-view.html...
I240729 05:14:21.439976 170 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/create-table-as.html...
I240729 05:14:21.449783 144 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.2/alter-type.html...
I240729 05:14:21.463422 171 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html...
I240729 05:14:21.469538 145 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html...
I240729 05:14:21.472030 172 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/show-columns.html...
I240729 05:14:21.477491 242 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/create-database.html...
I240729 05:14:21.478799 173 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/drop-table.html...
I240729 05:14:21.485045 243 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/explain.html...
I240729 05:14:21.491991 174 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/alter-partition.html...
I240729 05:14:21.494382 244 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/savepoint.html...
I240729 05:14:21.499645 175 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.2/show-create.html...
I240729 05:14:21.505958 245 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/set-vars.html...
I240729 05:14:21.507405 176 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/set-transaction.html...
I240729 05:14:21.512425 246 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html...
I240729 05:14:21.518648 177 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html...
I240729 05:14:21.522677 247 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.2/show-users.html...
I240729 05:14:21.525638 258 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/backup.html...
I240729 05:14:21.548302 248 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240729 05:14:21.576588 259 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/alter-table.html...
I240729 05:14:21.593403 260 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/revoke.html...
I240729 05:14:21.596348 249 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/create-index.html...
I240729 05:14:21.598089 250 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.2/create-view.html...
I240729 05:14:21.602198 274 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/create-table.html...
I240729 05:14:21.711656 291 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/show-vars.html...
I240729 05:14:21.735432 281 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html...
I240729 05:14:21.754681 292 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/show-backup.html...
I240729 05:14:21.759153 282 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/show-tables.html...
I240729 05:14:21.778475 293 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html...
I240729 05:14:21.778598 294 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/drop-function.html...
I240729 05:14:21.782384 267 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/alter-database.html...
I240729 05:14:21.787551 295 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/import-into.html...
I240729 05:14:21.789201 268 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.2/create-trigger.html...
I240729 05:14:21.799628 296 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html...
I240729 05:14:21.804075 269 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/delete.html...
I240729 05:14:21.827761 297 (gostd) urlcheck.go:235  [-] 56  Checking https://www.cockroachlabs.com/docs/v24.2/restore.html...
I240729 05:14:21.834311 270 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/grant.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             269:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             1344:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.98s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16258196?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16258196?buildTab=artifacts#/) on master @ [5d9405ff2e16622c055f94208992a5347c4c273e](https://github.com/cockroachdb/cockroach/commits/5d9405ff2e16622c055f94208992a5347c4c273e):


```
I240730 05:14:10.963125 68 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/alter-database.html...
I240730 05:14:10.963162 48 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/alter-function.html...
I240730 05:14:10.963200 47 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/show-backup.html...
I240730 05:14:10.963235 40 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/alter-type.html...
I240730 05:14:10.963298 71 (gostd) urlcheck.go:235  [-] 18  Checking https://www.cockroachlabs.com/docs/v24.2/alter-partition.html...
I240730 05:14:10.963398 69 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/drop-index.html...
I240730 05:14:10.963478 70 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.2/set-vars.html...
I240730 05:14:10.982551 227 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/create-trigger.html...
I240730 05:14:10.982732 141 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/show-vars.html...
I240730 05:14:10.983038 243 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.2/create-database.html...
I240730 05:14:10.983502 142 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.2/table-expressions.html...
I240730 05:14:10.984080 244 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/insert.html...
I240730 05:14:10.984807 143 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/create-index.html...
I240730 05:14:10.986891 245 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/explain.html...
I240730 05:14:10.987863 144 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/alter-index.html...
I240730 05:14:10.989693 246 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/import-into.html...
I240730 05:14:10.991118 145 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html...
I240730 05:14:10.992150 247 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/show-tables.html...
I240730 05:14:11.093158 258 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240730 05:14:11.093282 248 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/show-grants.html...
I240730 05:14:11.094838 228 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html...
I240730 05:14:11.096661 249 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/show-index.html...
I240730 05:14:11.097088 229 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/show-default-privileges...
I240730 05:14:11.097872 250 (gostd) urlcheck.go:235  [-] 37  Checking https://en.wikipedia.org/wiki/Cron...
I240730 05:14:11.099852 230 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html...
I240730 05:14:11.099941 259 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/show-databases.html...
I240730 05:14:11.101012 189 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/drop-database.html...
I240730 05:14:11.102179 260 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/create-function.html...
I240730 05:14:11.102733 190 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/drop-table.html...
I240730 05:14:11.102949 261 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html...
I240730 05:14:11.104193 191 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/truncate.html...
I240730 05:14:11.104546 231 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/select-clause.html...
I240730 05:14:11.104623 264 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.2/show-create.html...
I240730 05:14:11.110144 232 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/update.html...
I240730 05:14:11.111429 276 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html...
I240730 05:14:11.111986 233 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/alter-view.html...
I240730 05:14:11.115522 240 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/create-procedure.html...
I240730 05:14:11.115614 277 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/create-table.html...
I240730 05:14:11.116814 295 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/restore.html...
I240730 05:14:11.117232 267 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.2/grant.html...
I240730 05:14:11.119733 300 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.2/backup.html...
I240730 05:14:11.120049 305 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/show-ranges.html...
I240730 05:14:11.120742 273 (gostd) urlcheck.go:235  [-] 56  Checking https://www.cockroachlabs.com/docs/v24.2/show-columns.html...
I240730 05:14:11.121814 307 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             1272:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             374:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.31s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16270321?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16270321?buildTab=artifacts#/) on master @ [c7d1ceed692d36e9d7c5a71e7122408e2e3e3b8b](https://github.com/cockroachdb/cockroach/commits/c7d1ceed692d36e9d7c5a71e7122408e2e3e3b8b):


```
I240731 05:11:25.483772 25 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/drop-index.html...
I240731 05:11:25.483805 67 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html...
I240731 05:11:25.483848 29 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/alter-view.html...
I240731 05:11:25.483958 26 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/restore.html...
I240731 05:11:25.484045 27 (gostd) urlcheck.go:235  [-] 18  Checking https://www.cockroachlabs.com/docs/v24.2/show-columns.html...
I240731 05:11:25.484125 30 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/show-databases.html...
I240731 05:11:25.484211 28 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.2/show-default-privileges...
I240731 05:11:25.637442 219 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/alter-type.html...
I240731 05:11:25.639598 185 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/create-trigger.html...
I240731 05:11:25.640583 220 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.2/show-vars.html...
I240731 05:11:25.640894 186 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html...
I240731 05:11:25.645623 221 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/truncate.html...
I240731 05:11:25.646925 222 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240731 05:11:25.646963 187 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/create-table-as.html...
I240731 05:11:25.649569 223 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/update.html...
I240731 05:11:25.653989 188 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/show-users.html...
I240731 05:11:25.654811 224 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html...
I240731 05:11:25.655726 189 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/select-clause.html...
I240731 05:11:25.657017 225 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.2/table-expressions.html...
I240731 05:11:25.661990 190 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/create-procedure.html...
I240731 05:11:25.664702 191 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/show-grants.html...
I240731 05:11:25.674122 242 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/alter-table.html...
I240731 05:11:25.697761 157 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/grant.html...
I240731 05:11:25.705102 251 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.2/backup.html...
I240731 05:11:25.725101 158 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/show-create.html...
I240731 05:11:25.727388 252 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/drop-function.html...
I240731 05:11:25.739975 159 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/set-vars.html...
I240731 05:11:25.768282 160 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/show-index.html...
I240731 05:11:25.772422 253 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/show-tables.html...
I240731 05:11:25.774090 254 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.2/revoke.html...
I240731 05:11:25.776064 161 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html...
I240731 05:11:25.777096 294 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/show-ranges.html...
I240731 05:11:25.777226 276 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.2/delete.html...
I240731 05:11:25.780112 303 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/explain.html...
I240731 05:11:25.781884 278 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html...
I240731 05:11:25.783734 279 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html...
I240731 05:11:25.786387 314 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/import-into.html...
I240731 05:11:25.791082 315 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/upsert.html...
I240731 05:11:25.791458 280 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/alter-function.html...
I240731 05:11:25.794688 316 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.2/alter-partition.html...
I240731 05:11:25.823125 281 (gostd) urlcheck.go:235  [-] 54  Checking https://en.wikipedia.org/wiki/Cron...
I240731 05:11:25.825435 317 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/alter-database.html...
I240731 05:11:25.829455 320 (gostd) urlcheck.go:235  [-] 56  Checking https://www.cockroachlabs.com/docs/v24.2/savepoint.html...
I240731 05:11:25.831804 204 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/create-index.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             1279:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             1984:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.47s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16284959?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16284959?buildTab=artifacts#/) on master @ [7a1bf52daa3f3dcaa434fd66d2b3c960a3c9c2a5](https://github.com/cockroachdb/cockroach/commits/7a1bf52daa3f3dcaa434fd66d2b3c960a3c9c2a5):


```
I240801 05:11:43.574034 25 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240801 05:11:43.574087 27 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/table-expressions.html...
I240801 05:11:43.574162 29 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html...
I240801 05:11:43.574215 26 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/create-trigger.html...
I240801 05:11:43.574335 24 (gostd) urlcheck.go:235  [-] 18  Checking https://en.wikipedia.org/wiki/Cron...
I240801 05:11:43.574472 73 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/set-transaction.html...
I240801 05:11:43.574671 28 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.2/alter-view.html...
I240801 05:11:43.587845 242 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html...
I240801 05:11:43.699606 259 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/delete.html...
I240801 05:11:43.703997 232 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.2/show-users.html...
I240801 05:11:43.751053 260 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.2/explain.html...
I240801 05:11:43.787084 233 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html...
I240801 05:11:43.816964 251 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/select-clause.html...
I240801 05:11:43.822081 263 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/alter-partition.html...
I240801 05:11:43.829349 252 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html...
I240801 05:11:43.830970 264 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/drop-function.html...
I240801 05:11:43.844950 253 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/create-database.html...
I240801 05:11:43.852805 265 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/show-databases.html...
I240801 05:11:43.855839 254 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.2/show-tables.html...
I240801 05:11:43.858069 266 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html...
I240801 05:11:43.858381 255 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/alter-index.html...
I240801 05:11:43.864870 267 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/alter-table.html...
I240801 05:11:43.866240 256 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/alter-database.html...
I240801 05:11:43.872968 268 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.2/savepoint.html...
I240801 05:11:43.882995 257 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/show-default-privileges...
I240801 05:11:43.898086 269 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/drop-index.html...
I240801 05:11:43.910544 274 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/alter-function.html...
I240801 05:11:43.917994 270 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/drop-database.html...
I240801 05:11:43.919321 275 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/create-function.html...
I240801 05:11:43.964679 271 (gostd) urlcheck.go:235  [-] 43  Checking https://www.cockroachlabs.com/docs/v24.2/show-create.html...
I240801 05:11:44.000423 276 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/upsert.html...
I240801 05:11:44.009479 292 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/show-index.html...
I240801 05:11:44.011507 272 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.2/create-index.html...
I240801 05:11:44.036784 293 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/show-columns.html...
I240801 05:11:44.080015 273 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html...
I240801 05:11:44.082780 294 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/set-vars.html...
I240801 05:11:44.088369 306 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/backup.html...
I240801 05:11:44.088619 295 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/create-table.html...
I240801 05:11:44.090163 307 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/create-view.html...
I240801 05:11:44.099622 296 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.2/drop-table.html...
I240801 05:11:44.124411 308 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.2/show-backup.html...
I240801 05:11:44.134874 297 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/show-partitions.html...
I240801 05:11:44.137781 309 (gostd) urlcheck.go:235  [-] 56  Checking https://www.cockroachlabs.com/docs/v24.2/update.html...
I240801 05:11:44.162780 298 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/restore.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             1576:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             163:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.88s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16298788?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/16298788?buildTab=artifacts#/) on master @ [78fa126dfc749e620c28531c1659e35bf98034ec](https://github.com/cockroachdb/cockroach/commits/78fa126dfc749e620c28531c1659e35bf98034ec):


```
I240802 05:10:39.550181 40 (gostd) urlcheck.go:235  [-] 14  Checking https://www.cockroachlabs.com/docs/v24.2/show-tables.html...
I240802 05:10:39.550252 69 (gostd) urlcheck.go:235  [-] 15  Checking https://www.cockroachlabs.com/docs/v24.2/show-default-privileges...
I240802 05:10:39.550331 71 (gostd) urlcheck.go:235  [-] 16  Checking https://www.cockroachlabs.com/docs/v24.2/backup.html...
I240802 05:10:39.550391 70 (gostd) urlcheck.go:235  [-] 17  Checking https://www.cockroachlabs.com/docs/v24.2/import-into.html...
I240802 05:10:39.550513 46 (gostd) urlcheck.go:235  [-] 18  Checking https://www.cockroachlabs.com/docs/v24.2/drop-database.html...
I240802 05:10:39.550575 37 (gostd) urlcheck.go:235  [-] 19  Checking https://www.cockroachlabs.com/docs/v24.2/create-table.html...
I240802 05:10:39.550645 47 (gostd) urlcheck.go:235  [-] 20  Checking https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html...
I240802 05:10:39.715720 174 (gostd) urlcheck.go:235  [-] 21  Checking https://www.cockroachlabs.com/docs/v24.2/update.html...
I240802 05:10:39.753146 126 (gostd) urlcheck.go:235  [-] 22  Checking https://www.cockroachlabs.com/docs/v24.2/insert.html...
I240802 05:10:39.757696 261 (gostd) urlcheck.go:235  [-] 23  Checking https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html...
I240802 05:10:39.842506 127 (gostd) urlcheck.go:235  [-] 24  Checking https://www.cockroachlabs.com/docs/v24.2/upsert.html...
I240802 05:10:39.845881 262 (gostd) urlcheck.go:235  [-] 25  Checking https://www.cockroachlabs.com/docs/v24.2/show-partitions.html...
I240802 05:10:39.850606 128 (gostd) urlcheck.go:235  [-] 26  Checking https://www.cockroachlabs.com/docs/v24.2/alter-type.html...
I240802 05:10:39.851177 263 (gostd) urlcheck.go:235  [-] 27  Checking https://www.cockroachlabs.com/docs/v24.2/set-transaction.html...
I240802 05:10:39.856369 129 (gostd) urlcheck.go:235  [-] 28  Checking https://www.cockroachlabs.com/docs/v24.2/show-users.html...
I240802 05:10:39.860614 264 (gostd) urlcheck.go:235  [-] 29  Checking https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html...
I240802 05:10:39.862296 274 (gostd) urlcheck.go:235  [-] 30  Checking https://www.cockroachlabs.com/docs/v24.2/show-databases.html...
I240802 05:10:39.871480 265 (gostd) urlcheck.go:235  [-] 31  Checking https://www.cockroachlabs.com/docs/v24.2/drop-index.html...
I240802 05:10:39.878630 275 (gostd) urlcheck.go:235  [-] 32  Checking https://www.cockroachlabs.com/docs/v24.2/drop-table.html...
I240802 05:10:39.881397 266 (gostd) urlcheck.go:235  [-] 33  Checking https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html...
I240802 05:10:39.882747 276 (gostd) urlcheck.go:235  [-] 34  Checking https://www.cockroachlabs.com/docs/v24.2/table-expressions.html...
I240802 05:10:39.884112 267 (gostd) urlcheck.go:235  [-] 35  Checking https://www.cockroachlabs.com/docs/v24.2/show-create.html...
I240802 05:10:39.884508 277 (gostd) urlcheck.go:235  [-] 36  Checking https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html...
I240802 05:10:39.885687 268 (gostd) urlcheck.go:235  [-] 37  Checking https://www.cockroachlabs.com/docs/v24.2/create-function.html...
I240802 05:10:39.891108 278 (gostd) urlcheck.go:235  [-] 38  Checking https://www.cockroachlabs.com/docs/v24.2/alter-view.html...
I240802 05:10:39.898397 269 (gostd) urlcheck.go:235  [-] 39  Checking https://www.cockroachlabs.com/docs/v24.2/show-ranges.html...
I240802 05:10:39.947645 279 (gostd) urlcheck.go:235  [-] 40  Checking https://www.cockroachlabs.com/docs/v24.2/savepoint.html...
I240802 05:10:39.986575 270 (gostd) urlcheck.go:235  [-] 41  Checking https://www.cockroachlabs.com/docs/v24.2/create-index.html...
I240802 05:10:39.995022 280 (gostd) urlcheck.go:235  [-] 42  Checking https://www.cockroachlabs.com/docs/v24.2/truncate.html...
I240802 05:10:39.998989 271 (gostd) urlcheck.go:235  [-] 43  Checking https://en.wikipedia.org/wiki/Cron...
I240802 05:10:40.010710 243 (gostd) urlcheck.go:235  [-] 44  Checking https://www.cockroachlabs.com/docs/v24.2/alter-index.html...
I240802 05:10:40.018623 296 (gostd) urlcheck.go:235  [-] 45  Checking https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html...
I240802 05:10:40.069747 244 (gostd) urlcheck.go:235  [-] 46  Checking https://www.cockroachlabs.com/docs/v24.2/alter-table.html...
I240802 05:10:40.088042 297 (gostd) urlcheck.go:235  [-] 47  Checking https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html...
I240802 05:10:40.118681 245 (gostd) urlcheck.go:235  [-] 48  Checking https://www.cockroachlabs.com/docs/v24.2/show-vars.html...
I240802 05:10:40.123235 298 (gostd) urlcheck.go:235  [-] 49  Checking https://www.cockroachlabs.com/docs/v24.2/select-clause.html...
I240802 05:10:40.126936 246 (gostd) urlcheck.go:235  [-] 50  Checking https://www.cockroachlabs.com/docs/v24.2/create-table-as.html...
I240802 05:10:40.127731 299 (gostd) urlcheck.go:235  [-] 51  Checking https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html...
I240802 05:10:40.133695 247 (gostd) urlcheck.go:235  [-] 52  Checking https://www.cockroachlabs.com/docs/v24.2/alter-partition.html...
I240802 05:10:40.140322 300 (gostd) urlcheck.go:235  [-] 53  Checking https://www.cockroachlabs.com/docs/v24.2/delete.html...
I240802 05:10:40.141092 248 (gostd) urlcheck.go:235  [-] 54  Checking https://www.cockroachlabs.com/docs/v24.2/show-columns.html...
I240802 05:10:40.152505 301 (gostd) urlcheck.go:235  [-] 55  Checking https://www.cockroachlabs.com/docs/v24.2/show-grants.html...
I240802 05:10:40.153280 249 (gostd) urlcheck.go:235  [-] 56  Checking https://www.cockroachlabs.com/docs/v24.2/show-constraints.html...
I240802 05:10:40.159294 302 (gostd) urlcheck.go:235  [-] 57  Checking https://www.cockroachlabs.com/docs/v24.2/show-backup.html...
    nightly_lint_test.go:46: https://www.cockroachlabs.com/docs/v24.2/create-trigger.html : 404 Not Found
             50:  https://www.cockroachlabs.com/docs/v24.2/create-trigger.html
        https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html : 404 Not Found
             2147:  https://www.cockroachlabs.com/docs/v24.2/drop-trigger.html
        2 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.93s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/sql/parser/sql.y
+++ b/pkg/sql/parser/sql.y
@@ -5391,7 +5391,6 @@ opt_no:
 //  [ FOR { EACH ROW | EACH STATEMENT } ]
 //  [ WHEN a_expr ]
 //  EXECUTE FUNCTION func_name ( trigger_func_args )
-// %SeeAlso: WEBDOCS/create-trigger.html
 create_trigger_stmt:
   CREATE opt_or_replace TRIGGER name trigger_action_time trigger_event_list
   ON table_name opt_trigger_transition_list trigger_for_each trigger_when
@@ -5578,7 +5577,6 @@ trigger_func_arg:
 // %Category: DDL
 // %Text:
 // DROP TRIGGER [ IF EXISTS ] name ON table_name [ CASCADE | RESTRICT ]
-// %SeeAlso: WEBDOCS/drop-trigger.html
 drop_trigger_stmt:
   DROP TRIGGER name ON table_name opt_drop_behavior
   {

